cli_tools/common/utils/logging/service/log_entry.go

```diff
@@ -217,6 +217,12 @@ type OutputInfo struct {
 	// Serial output from worker instances; only populated
 	// if workflow failed.
 	SerialOutputs []string `json:"serial_outputs,omitempty"`
+	// Shadow disk match result for shadow disk inflater
+	ShadowDiskMatchResult string `json:"shadow_disk_match_result,omitempty"`
+	// Inflation time (seconds)
+	InflationTime []int64 `json:"inflation_time,omitempty"`
+	// Inflation time (seconds) of the shadow disk
+	ShadowInflationTime []int64 `json:"shadow_inflation_time,omitempty"`
```

Comment:
1. Currently only string and []int64 are supported by loggable.GetValues. It's not worth extracting the 1st element since it's harmless to use array here.
2. The array can be empty, but a int64 can't. We don't want a "0" value while it actually doesn't exist.